tools/block-generator/runner/run.go

```diff
@@ -166,6 +168,12 @@ func (r *Args) run(reportDirectory string) error {
 		}
 		fmt.Printf("%sPostgreSQL next round: %d\n", pad, nextRound)
 	}
+
+	if r.StartDelay > 0 {
```

Comment:
Does this wait even before the first run? If so, is this desirable for a specific reason - i.e. we want a breather after completing the snapshot operation? Or perhaps a different variable `IntermissionDelay` would be preferred?

Reply:
Yes, this is intentional. The first run after restoring a snapshot is much slower than any of the other tests. In the deviation column you added, this is the only time between run1/run2 where the difference is positive.

Reply:
The TPS for **run2** decreased an average 17% in comparison with run1 _after a snapshot_. So it seems that **run2** needs a cool down period more than **run1** does. But regardless, it seems reasonable to delay after the special snapshot condition - approving.